test/laguerre-gauss.jl

```diff
@@ -4,6 +4,8 @@ using StaticArrays
 using UnitfulAtomic
 using LinearAlgebra
 
+pochhammer(a::Int,b::Int) = factorial(a+b-1)/factorial(a-1)
```

Comment:
You can use the function defined in `LaserTypes` with `using LaserTypes: pochammer`